CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Breaking changes result in a different major. UI changes that might break custom
 
 ## [next-version]
 
+## [0.13.0-rc.1]
+
+Install with `npm install onfido-sdk-ui@0.13.0-rc.1`
```

Comment:
For release candidates you need to include the specific tag `@0.13.0-rc.1`

`npm install onfidp-sdk-ui` installs the latest untagged version so would not install this version.